Engineering.md

```diff
@@ -79,47 +93,43 @@
 - I am actively introspective and take time to work on my personal and professional development.
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 
+## Team Lead
+
+> I am trusted to autonomously lead people through complex problems with a proven record of delivering ideas from inception to their successful completion.
+> A Team Lead emcompasses all of the qualities listed in [Leadership](Leadership.md) and is usually combined with either a Senior Software Engineer or Solution Architect role.
+
+### I am trusted to lead a team of diverse technical roles (from Architects, Engineers, Designers and Testers) to ship full products to production
```

Comment:
What's an example of a "full product" here? Should this be "shaped pitch"?

Reply:
Yes! Missed one, ty